binding.gyp

```diff
@@ -1,7 +1,7 @@
 {
     "targets": [
         {
-            "target_name": "fs-ext", 
+            "target_name": "fs_ext", 
```

Comment:
Note: NODE_MODULE_INIT uses NODE_GYP_MODULE_NAME as an identifier, so this must be a valid identifier.